class Exception2 
{
	public static void main(String[] args) 
	{
		try
		{
			
		}
		catch (Exception e)
		{
		}
		catch (ArithmeticException ae)
		{

		}
		catch (NullPointerException npe)
		{

		}
		
	}
}
